File: Python/sumar.py
# sumar.py
import sys

# Capturar argumentos de la línea de comandos
try:
    num1 = int(sys.argv[1])
    num2 = int(sys.argv[2])
    print(f"La suma de {num1} y {num2} es {num1 + num2}")
except (IndexError, ValueError):
    print("Por favor, proporciona dos números como argumentos.")

#docker run --rm -v "$(pwd):/app" -w /app python:3.11 python sumar.py 5 7

File: NodeJS/sumar.js
const readline = require('readline');

// Crear interfaz para leer entrada del usuario
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// Preguntar por el primer número
rl.question('Ingresa el primer número: ', (num1) => {
  // Preguntar por el segundo número
  rl.question('Ingresa el segundo número: ', (num2) => {
    // Intentar convertir las entradas a números y sumar
    const suma = parseFloat(num1) + parseFloat(num2);

    if (isNaN(suma)) {
      console.log('Por favor, ingresa valores numéricos válidos.');
    } else {
      console.log(`La suma de ${num1} y ${num2} es ${suma}`);
    }

    // Cerrar la interfaz
    rl.close();
  });
});

//docker run -it --rm -v "$(pwd):/app" -w /app node-dev bash
